assets/airlines/klc.json

```diff
@@ -2,14 +2,15 @@
     "icao": "klc",
     "name": "KLM Cityhopper",
     "callsign": {
-        "name": "KLM",
+        "name": "City",
```

Comment:
Change this back to KLM. They never use the "City" callsign.

Reply:
They never use it because they never fly with the `KLC` icao code, they are always operating for the normal `KLM`. This airline should't be used actually, it's more realistic to just use the "embraers" fleet of the normal KLM airline file.
![](https://i.imgur.com/IJeJkYI.png)

Reply:
Yes, so the option is either change it, keep it, or delete it